apps/src/music/player/MusicLibrary.ts

```diff
@@ -1,12 +1,15 @@
+
```

Comment:
nit: probably just need to manually run linting on this file (since we don't have linting set up for TS currently)

Reply:
Ah, indeed.